[doc][yba][2.20.3] CSP OS patching (#21449)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md

  For information on configuring your regions, see [Add regions](#add-regions).

### Linux version catalog

Specify the machine images to be used to install on nodes of universes created using this provider.

To add machine images recommended and provisioned by YBA, select the **Include Linux versions that are chosen and managed by YugabyteDB Anywhere in the catalog** option, and choose the architectures.

To add your own machine images to the catalog:

1. Click **Add Linux Version**.

> 1. Provide a name for the Linux version. User will see this name when creating universes using this provider.

Reviewer: ```suggestion
1. Provide a name for the Linux version. You can see this name when creating universes using this provider.
```
